components/camel-undertow/src/main/java/org/apache/camel/component/undertow/UndertowComponent.java

```diff
@@ -18,11 +18,7 @@
 
 import java.net.URI;
 import java.net.URISyntaxException;
-import java.util.HashMap;
-import java.util.LinkedHashSet;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Is the checkstyle enabled? That shouldn't be there

Reply:
I'll fix it. I've run mvn checkstyle:check before commit without an error...

Reply:
Is it the same as ` mvn clean install -Psourcecheck`?

Reply:
I hope so, but I'll try it. Both occurrences are fixed.